Client side crud

MTse3 opened the pull request:
angular crud with no auth
still need to edit cancel buttons

MTse3 committed:
can make new full new players

MTse3 committed:
still needs to edit cancel buttons

MTse3 committed:
some changes to work through cancel button

MTse3 committed:
successful karma test

MTse3 committed:
added empty css files
adjusted gulpfile for css
added working test for getAll();
almost working post test

MTse3 committed:
added flexbox properties for more responsive layout
minified css to be included in html
adjusted html to containerize elements

MTse3 committed:
added a gulp watch
adjusted some testing
issues with build
added some services that could potentially be implemented

MTse3 committed:
changed name in players_controllers_tests to check if bundle was built

MTse3 committed:
gulp watch  work, attemmpted to flex li oflist

MTse3 committed:
resolved issue with getting players by team name in player routes
player tests all pass again

MTse3 committed:
stored database data into an object for cancel edit to work

MTse3 committed:
took away an unneeded commented out area

MTse3 committed:
added sass to app
changed gulpfile to remove concat
all css files changed to scss
scss files imported into app.scss
index requires app.scss

MTse3 committed:
centered heading
attempted to adjust layout

warmbowski commented on michael_tse/app/js/players/controllers/players_controller.js:
```diff
@@ -0,0 +1,63 @@
+        });
+    };
+
+    $scope.update = function(player) {
```
Two $scope.update methods? I don't think so.

MTse3 committed:
added services to get and post requests
added dummy directive
issue with not being able to find template html file

MTse3 committed:
removed some commented out code that was replaced by services

MTse3 committed:
made service for update

MTse3 committed:
made services for delete request
crud is now in services
cleaned up excess comments

MTse3 committed:
spacing issues in player_controller

MTse3 committed:
Merge branch 'directives' into sass

MTse3 committed:
changed layout to be a skeleton grid for responsiveness
made options for right left batting and throwing
numbers input takes number value
stylized title

MTse3 committed:
declared colors as variables
set defaults for colors and button size/radius
stylized button and heading using declared variables in base scss

MTse3 committed:
increased heading size

MTse3 committed:
changed server file for express static to find build files
added sample directives for transclude  and forms
file structure works, need to adjust for content
commented out parts of html to potentially use as directive

MTse3 committed:
refactored to have directive of forms be in two places (create and edit)
added an player edit template for editing a player (can be removed after cancel button works)
moved bulk of main into players_view html
needs to fix issue with cancel button not showing

MTse3 committed:
transcluded cancel edit button
removed edit templates(redundant with form directive)
fixed typo with player's position not showing

MTse3 committed:
added function directive as part of player form directive to default button text to be create new player
created table to display all the bio stats for the players in the database

MTse3 committed:
Merge pull request #10 from MTse3/styles

Styles Css

MTse3 committed:
changed layout to include skeleton to match sass

MTse3 committed:
Merge pull request #13 from MTse3/directives-more

more directives and table added

MTse3 committed:
Merge branch 'client-side-crud' of https://github.com/MTse3/mongo_backed_rest_api into client-side-crud

MTse3 committed:
merged from sass to remove css files to make sass
also merged directives (from sass branch) to be more updated
